server/views/pages/induction/checkYourAnswers/partials/_abilityToWork.njk

```diff
@@ -7,7 +7,7 @@
     </dt>
     <dd class="govuk-summary-list__value" data-qa="abilityToWork">
       {% for item in inductionDto.workOnRelease.affectAbilityToWork %}
-        <div data-qa="skills-{{ loop.index }}">{{ item | formatAbilityToWorkConstraint }}{{ ' - ' + inductionDto.workOnRelease.affectAbilityToWorkOther if item === 'OTHER' }}{{ ',' if inductionDto.workOnRelease.affectAbilityToWork.length !== loop.index }}</div>
+        <div data-qa="affectingAbilityToWork-{{ item }}">{{ item | formatAbilityToWorkConstraint }}{{ ' - ' + inductionDto.workOnRelease.affectAbilityToWorkOther if item === 'OTHER' }}{{ ',' if inductionDto.workOnRelease.affectAbilityToWork.length !== loop.index }}</div>
```

Comment:
In line with your previous change, I've updated some of these loop indexes to use keys for testing.